src/relay/op/call/call.cc

```diff
@@ -95,20 +96,29 @@ RELAY_REGISTER_OP("call_lowered")
     .set_attr<FInferCorrectLayout>("FInferCorrectLayout", ElemwiseArbitraryLayout);
 
 CallLoweredProps GetCallLoweredProps(const CallNode* call_node) {
```

Comment:
Given this is such a fundamental function it seems like we should have test cases for all the different cases it throws an exception and what we expect as a return value. Though I equally understand this wasn't authored in this PR.

@mbs-octoml / @electriclilies can you take a look at this?

Reply:
Thanks @Mousius, agree and sorry I missed that in the orig PR. Can you take that @electriclilies ? Thanks!

Reply:
Yup!